src/number_format.js

```diff
@@ -78,6 +78,7 @@ const defaultProps = {
   onBlur: noop,
   isAllowed: returnTrue,
   getInputRef: noop,
+  accounting: false,
```

Comment:
Define it on proptype as well. Also I think the prop name should be `negationFormat` whose value can be 'parentheses | minus'  default to minus.